src/python/pants/build_graph/build_file_address_mapper.py

```diff
@@ -253,7 +253,7 @@ def _raise_incorrect_address_error(self, spec_path, wrong_target_name, addresses
 
     if not addresses:
       raise self.EmptyBuildFileError(
-        '{was_not_found_message}, because that directory contains no BUILD files defining addressable entities.'
+        '{was_not_found_message}, because that directory does not contain any BUILD files defining addressable entities.'
```

Comment:
This feels like it's unrelated to this change, but I do like the updated wording better.

Reply:
It's related to the change because a bunch of tests are swapping from being run in `v1` to being run in `v2`.